docs/fsharp/language-reference/reference-cells.md

```diff
@@ -101,6 +98,45 @@ The following code examples demonstrate the use of reference cells in closures.
 
 [!code-fsharp[Main](../../../samples/snippets/fsharp/lang-ref-1/snippet2207.fs)]
 
+## Consuming C#7 and VB15 `ref` returns
```

Comment:
Thinking about it, this makes sense.  I also removed traces of their version numbers.